.pre-commit-config.yaml

```
@@ -17,6 +17,11 @@ repos:
     rev: v0.3.1
     hooks:
       - id: absolufy-imports
+  - repo: https://github.com/econchick/interrogate
+    rev: 1.5.0
+    hooks:
+      - id: interrogate
+        args: [-viImMpPsSCn, -f, "25.0", merlin]
```

When I was push out https://github.com/NVIDIA-Merlin/core/pull/44 - I noticed that the pre-commit hook with interrogate was failing the change - and I had to disable the pre-commit hook to commit the change.  

It appears that the interrogate pre-commit hook was including files from the tests/ directory (which you can verify by running `pre-commit run --all-files` on merlin-core) . Changing the pyproject.toml file didn't seem to work, nor did passing the '--exclude=tests' to the comamndline here.

This does seem to work though (like tell pre-commit to ignore tests rather than tell interrogate to ignore tests)

```suggestion
        args: [-viImMpPsSCn, -f, "25.0", merlin]
        exclude: tests
```

---

Huh, that's interesting. My impression was that it only checks the supplied directory (`merlin`.)

---

It sorta looked like pre-commit was passing a list of all the changed files to interrogate - which was overriding the other settings. Like it was running interrogate on the merlin directory, and the changed tests (rather than just the merlin directory) - but not on the tests that I didn't touch

---

Ahhhh, that makes sense